samples/LocalizationPlayground/CultureEnvironmentCommandLineExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.CommandLine.Builder;
+using System.CommandLine.Invocation;
+using System.Globalization;
+using System.Threading;
+
+namespace LocalizationPlayground
+{
+    internal static class CultureEnvironmentCommandLineExtensions
+    {
+        internal static CommandLineBuilder UseCultureEnvironment(
+            this CommandLineBuilder builder)
+        {
+            return builder.UseMiddleware(async (context, next) =>
+            {
+                if (Environment.GetEnvironmentVariable("DOTNET_SYSTEM_GLOBALIZATION_CULTURE") is string culture &&
+                        !string.IsNullOrEmpty(culture))
+                    CultureInfo.CurrentCulture = CultureInfo.CurrentUICulture = CultureInfo.GetCultureInfo(culture);
+
+                var execCtx = ExecutionContext.Capture();
```

Comment:
Is there an approach that can work without using `ExecutionContext`? It's fairly heavy in terms of performance.

Reply:
Yeah, this is just a quick and dirty hack to achieve this in the playground, but I am using the same trick in #951, so we should discuss it there... When we merge #951 this won't be necessary anyways. In the meantime is it okay to just keep this here, since it's just the playground?

Reply:
@jonsequitur I improved it slightly in 8587a5c3